src/app/zap-templates/partials/idl/cluster_attribute.zapt

```diff
@@ -0,0 +1,26 @@
+  {{! ensure indent}}{{#unless isWritable~}}
+  readonly {{!marker to place a space even with whitespace removal~}}
+{{~/unless~}}
+{{~!TODO: write only attributes should also be supported~}}
```

Comment:
This is supported by `isWritableAttribute` and `isWritable` under zcl_attributes_server

Reply:
No, this is about _write only_ attributes.  That is, attributes that are writable but not readable.  ZAP has no way to represent those right now.